### PR TITLE
fix: responsive author avatar

### PR DESCRIPTION
**Pull request by fampinheiro**
## desktop

image

## tablet

image

## mobile

image

### CODE REVIEW

File: packages/author-head/__tests__/author-head.test.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ it("tracks twitter link interactions", () => {
 
   component
     .dive()
+    .dive()
```
Comment (damnhipster) [inline packages/author-head/__tests__/author-head.test.js]: dive²

**Comment by trevorah** [comment]
I'm not the biggest fan of having a responsive component like this.

I'd rather have:
```js
withWindowBreakpoints(Component, { size: 768, component: ComponentA },  { size: 1024, component: ComponentB })
```

File: packages/author-head/__tests__/author-head.test.js (hunk 1 above)
Comment (fampinheiro) [reply packages/author-head/__tests__/author-head.test.js]: With the tracking HoC we need to dive deeper to access the links

**Comment by fampinheiro** [comment]
@trevorah, in that case, we need different components for different screen sizes, right?

**Comment by craigbilner** [comment]
needs a rethink

**Pull request closed without merging**